src/dataset_tools/bq_squashfs/download_images.py

```
@@ -0,0 +1,627 @@
+
+        # Inline MERGE into training_images so status is current without a separate job
+        n_updated = merge_chunk_into_training_images(
+            client, results, training_table, downloads_table
+        )
+        print(f"  Merged {n_updated:,} rows into {training_table}", flush=True)
+
+        # Pack images into chunk sqfs then clear raw files to keep inode usage low
+        pack_chunk_to_sqfs(staging_dir, chunk_num, num_workers=4)
+        clear_staging(staging_dir)
+        warn_chunk_accumulation(staging_dir)
+        print(f"  Staging cleared (chunk sqfs kept)", flush=True)
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)